update 'Features' and 'What We Are Working On' sections in README

patrycja-gomza opened the pull request:
### Description of Changes
Updated the 'Features' and 'What We Are Working On' sections in the README file to reflect the latest status and progress of the project. These changes were made in preparation for the Monday project review with our mentor.

### Next Steps for README.md
For the next update of the README.md file, I plan to include a GIF showcasing all the functionalities of the application, along with screenshots capturing the error and loading states. 
I'm open to any suggestions or additional content that the team may propose 😊

Thank you for your collaboration! 🙂

bedicooper reviewed (approved):
Thanks for great addition, of course I approve it.

As we forgot to maybe its a good time to also add a section in `## Available Scripts` about `npm run deploy` script that we've added? 

Also since we just used a new library, maybe we could split `## Technologies` and add `### Libraries` sction, with react, react-dom, react-spring, redux with RTK, axios, Styled-Comopnents etc?

I also think that we it would be nice to provide information how to use our code when somebody clones the repository, mainly how to handle the API - where to get the key from and where in our repo to change it.

patrycja-gomza commented:
Hi @bedicooper 👋

Thank you for your feedback! I have already addressed the first two suggestions:
- Added a section about the npm run deploy script in the ## Available Scripts.
- Split ## Technologies into multiple sections, including ### Libraries, Technologies and Tools, Build and Deployment.

Regarding the third issue related to the API section, I will work on providing the missing information in the upcoming pull request.

Thank you for your valuable feedback and support! 🙏

bedicooper reviewed (approved):
OK, thanks for taking my suggestions into account 🙏
